CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 ##### Bug Fixes
 
-* None
+ * Only update contentInset if the new insets are different from the existing insets. This avoids triggering unnecessary table/collectionview reloads.
```

Comment:
Might just be me, but the spacing seems off here?

Reply:
Whoops, fixed.